### DISCUSSION TITLE
Is it possible to have stars from a project you were a contributor to, reflected on your personal profile?

### DISCUSSION THREAD

**Discussion by JustinWouters** [community/community#27651] [Other Feature Feedback, Questions, & Ideas]
I recently was part of a small team that created an Open Source Project and it gained a good amount of popularity from the community and received a decent amount of stars. I am wondering since the repository was created under a GitHub account that was made specifically for this project if there is any way for my team and I to have the stars that the project received reflected on our own GitHub profiles?

**Comment by DmitriyShum** [comment]
Hello, the amount of stars on a different account's repository will not be reflected on your profiles since you're not the owner of that repository.

**Comment by jfullstackdev** [comment]
if you want to display it right there on your landing README, there is an external service to do that, simply paste this code on your README 

```
[![Readme Card](https://github-readme-stats.vercel.app/api/pin/?username=open-source-labs&repo=Svelvet&theme=tokyonight)](https://github.com/open-source-labs/Svelvet)
```

[![Readme Card](https://github-readme-stats.vercel.app/api/pin/?username=open-source-labs&repo=Svelvet&theme=tokyonight)](https://github.com/open-source-labs/Svelvet)

for more info about that service,

https://github.com/anuraghazra/github-readme-stats
